src/subvariable.jl

```diff
@@ -1,73 +1,68 @@
 
-Base.parent(v::SubVariable) = v.parent
-Base.parentindices(v::SubVariable) = v.indices
-Base.size(v::SubVariable) = _shape_after_slice(size(parent(v)),v.indices...)
+function Base.show(io::IO, v::SubVariable)
+    level = get(io, :level, 0)
+    indent = " " ^ get(io, :level, 0)
+    delim = " × "
+    try
+        indices = parentindices(v)
+        print(io,indent,"View:  ",join(indices,delim),"\n")
```

Comment:
Should there be some colour here?

Reply:
feel free to add :-) (but it is also ok if there is no color)